app/Http/Controllers/Api/SecurityAuthController.php

```
@@ -0,0 +1,171 @@
+                'phone' => $personnel->phone,
+                'organization_type' => $personnel->organization_type,
+                'organization_name' => $personnel->organization_name,
+                'role' => 'security_' . $personnel->organization_type,
+            ],
+        ]);
+    }
+
+    /**
+     * Get current authenticated security personnel
+     */
+    public function me(Request $request): JsonResponse
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)